### PR TITLE
fixed typos etc

### PR DESCRIPTION
**Pull request by ErbB4**

### CODE REVIEW

File: doc/python/simulation.rst [hunk 1]
```diff
@@ -34,7 +34,7 @@ over the local and distributed hardware resources (see :ref:`pydomdec`). Then, t
         # Instantiate the simulation.
         sim = arbor.simulation(recipe, decomp, context)
 
-        # Run the simulation for 2000 ms with time stepping of 0.025 ms
+        # Run the simulation for 2000 ms with a time step of 0.025 ms
```
Comment (thorstenhater) [inline doc/python/simulation.rst:37]: These should have units ;)

File: doc/python/simulation.rst [hunk 2]
```diff
@@ -230,16 +230,16 @@ Spikes recorded during a simulation are returned as a NumPy structured datatype
...
         sim.record(arbor.spike_recording.all)
 
-        # Run the simulation for 2000 ms with time stepping of 0.025 ms
+        # Run the simulation for 2000 ms with a time step of 0.025 ms
         tSim = 2000
         dt = 0.025
         sim.run(tSim, dt)
 
-        # Print the spikes and according spike time
+        # Print the spikes and accordingly the spike time
```
Comment (thorstenhater) [inline doc/python/simulation.rst:242]:
```suggestion
        # Print the spikes: time and source 
```

File: doc/python/simulation.rst [hunk 3]
```diff
@@ -230,16 +230,16 @@ Spikes recorded during a simulation are returned as a NumPy structured datatype
         sim = arbor.simulation(recipe, decomp, context)
 
         # Direct the simulation to record all spikes, which will record all spikes
-        # across multiple MPI ranks in distrubuted simulation.
+        # across multiple MPI ranks in distributed simulation.
         # To only record spikes from the local MPI rank, use arbor.spike_recording.local
         sim.record(arbor.spike_recording.all)
 
-        # Run the simulation for 2000 ms with time stepping of 0.025 ms
+        # Run the simulation for 2000 ms with a time step of 0.025 ms
```
Comment (thorstenhater) [inline doc/python/simulation.rst:237]: Same here, units missing. Again, not your fault, but let's do it right

**Review by thorstenhater — CHANGES_REQUESTED** [review]

File: doc/python/simulation.rst [hunk 4]
```diff
@@ -288,27 +288,27 @@ There are three parts to the process of recording cell data over a simulation.
...
     # [... define recipe, decomposition, context ... ]
     # Initialize simulation:
 
-    sim = arbor.simulation(recipe, decomp, context)
+    sim = A.simulation(recipe, decomp, context)
 
     # Sample probeset id (0, 0) (first probeset id on cell 0) every 0.1 ms
 
-    handle = sim.sample((0, 0), arbor.regular_schedule(0.1))
+    handle = sim.sample((0, 0), A.regular_schedule(0.1))
```
Comment (thorstenhater) [inline doc/python/simulation.rst:311]: unit!

**Review by thorstenhater — CHANGES_REQUESTED** [review]

File: doc/python/simulation.rst (hunk 4 above)
Comment (ErbB4) [reply doc/python/simulation.rst:311]: okkkkkkk

File: doc/python/simulation.rst [hunk 5]
```diff
@@ -288,31 +288,32 @@ There are three parts to the process of recording cell data over a simulation.
...
```
Comment (thorstenhater) [inline doc/python/simulation.rst:312]:
```suggestion
    handle = sim.sample((0, 0), A.regular_schedule(0.1*U.ms))
```

File: doc/python/simulation.rst [hunk 6]
```diff
@@ -224,22 +224,22 @@ Spikes recorded during a simulation are returned as a NumPy structured datatype
...
-        sim.record(arbor.spike_recording.all)
+        # across multiple MPI ranks in distributed simulation.
+        # To only record spikes from the local MPI rank, use A.spike_recording.local
+        sim.record(A.spike_recording.all)
 
-        # Run the simulation for 2000 ms with time stepping of 0.025 ms
-        tSim = 2000
-        dt = 0.025
+        # Run the simulation for 2000 ms with a time step of 0.025 ms
+        tSim = 2000 * U.ms
```
Comment (thorstenhater) [inline doc/python/simulation.rst:238]: On a pure technicality: You _could_ write
```suggestion
        tSim = 2 * U.s
```
instead

**Review by thorstenhater — CHANGES_REQUESTED** [review]

**Review by thorstenhater — APPROVED** [review]

**Pull request merged**